backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -34,9 +47,21 @@ def reshape_for_curation_api_and_is_allowed(collection, token_info, uuid_provide
     if "datasets" in collection:
         for dataset in collection["datasets"]:
             if "artifacts" in dataset:
```

Comment:
looks like we're no longer removing the "artifacts" entry from datasets--not seeing it in the OpenAPI spec, should we remove after this block?